.github/workflows/test.yml

```diff
@@ -290,7 +291,7 @@ jobs:
         brew install --cask xquartz
         brew install llvm ninja open-mpi
         # TTK dependencies
-        brew install boost eigen graphviz spectra sqlite zlib numpy qhull
+        brew install boost cgal eigen graphviz spectra sqlite zlib numpy qhull
```

Comment:
!please also add that dependency in the `package.yml`

NOTE: I tried to push this change myself as I'd usually do but I believe you uncheck the box "allow maintainer to push edits" or something like that (thanks for checking that box in the future or add me as a collaborator of your public fork. Thanks!)